### ISSUE TITLE
am I the only one?

### ISSUE THREAD

**Issue by ghost** [hmarr/mongoengine#454]
In my project, I heavily use mongoengine. 

But.. for some reason, whenever I install Mongoengine and run it, always an exception is raised..

So I have to change mongoengine's code..

The raised part of code is session.py 44 lines

```
self.session_key = self._get_new_session_key()
```

but the problem is that.. 

self.session_key property has no "set" functionality that allow session_key to be assigned by some value as you can see it

```
session_key = property(_get_session_key)
```

as you can see, there is no set method.

I'm not sure I'm correct since I think this is the only problem that I only have and other people don't.

but it'll be better for me to change it to something like this.. one day.. just hope it.. so that I don't have to change Mongoengine's code..

in session.py 44 lines..

```
 self._session_key = self._get_new_session_key()
```

or session_key has set property .. something like this one..

```
session_key = property(_get_session_key, _set_session_key)
```

anyway, IMO, Mongoengine is really great!!!!

**Comment by rozza** [comment]
Hi is this in the Django code?  What version of Django are you running?

**Comment by ghost** [comment]
it's located in 
C:\Python27\Lib\site-packages\mongoengine-0.6-py2.7.egg\mongoengine\django\session.py line 44 
Here you can see this code
    self.session_key = self._get_new_session_key()

so this code is Mongoengine

and.. the other thing is Django Code 1.4beta

C:\Python27\Lib\site-packages\django\contrib\sessions\backends\base.py 166 line
session_key = property(_get_session_key)

My django Version... 1.4 beta

> > > django.VERSION
> > > (1, 4, 0, 'beta', 1)

**Comment by rozza** [comment]
Ok thought so.  This is a dupe and will be fixed in 0.7

**Comment by ghost** [comment]
oh~~~ cool!!! This is the very first participation in my whole life!!
